FreeRTOS-Plus/Demo/coreHTTP_Windows_Simulator/HTTP_S3_Download/demo_config.h

```diff
@@ -207,7 +207,7 @@ extern void vLoggingPrintf( const char * pcFormatString,
 /**
  * @brief Transport timeout in milliseconds for transport send and receive.
  */
-#define democonfigTRANSPORT_SEND_RECV_TIMEOUT_MS    ( 5000 )
+#define democonfigTRANSPORT_SEND_RECV_TIMEOUT_MS    ( 10000 )
```

Comment:
The 5 second timeout was causing the demo to fail when using debug printing, bump it to 10 second timeout to create a more forgiving experience.